src/host/outputStream.cpp

```diff
@@ -76,8 +76,13 @@ void WriteBuffer::_DefaultStringCase(const std::wstring_view string)
 {
     size_t dwNumBytes = string.size() * sizeof(wchar_t);
 
-    _io.GetActiveOutputBuffer().GetTextBuffer().GetCursor().SetIsOn(true);
+    Cursor& cursor = _io.GetActiveOutputBuffer().GetTextBuffer().GetCursor();
+    if (!cursor.IsOn())
+    {
+        cursor.SetIsOn(true);
+    }
 
+    //cursor.StartDeferDrawing();

```

Comment:
I'm no sure if this is a safe change, though.

This reduces the unnecessary cursor redrawing caused by `WriteCharsLegacy` where a lot of cursor position adjustment happens.

Reply:
Why wouldn't this be a safe change?

Reply:
TBH it's because I'm afraid of `WriteCharsLegacy`, one of the most easy-to-break piece of code in conhost.

Reply:
No, do it. Please. It honestly was deferred like 7 years ago and I think I screwed it up and it's how I've been grinding out "cursor turds" bugs ever since. And if I'm wrong.... well it won't be the first or last time.